src/controllers/recentlyListenedController.js

```diff
@@ -0,0 +1,19 @@
+const RecentlyListened = require('../models/RecentlyListened');
+
+const getRecentlyListenedAlbumsForUser = async (req, res) => {
+  try {
+    const userId = req.params.userId;
+
+    // Fetch recently listened albums for the user, sorted by time in descending order
+    const recentlyListened = await RecentlyListened.find({ user: userId })
+      .sort({ timeListened: -1 })
+      .limit(10); // set the limit to 10 albums
```

Comment:
Looks good; but I do wonder if the frontend/caller will want more than just the user id + album id, maybe would be good to populate some additional fields (ie: user name, album name, album artist) as part of the response.

Currently response looks like:

```
[
  {
    "_id": "665f6d32eabaab4cebdaff45",
    "user": "64d6a966a15d2e18ab96a263",
    "album": "64d2ae2308a725b72bd5c0cf",
    "timeListened": "2024-06-04T19:38:26.917Z",
    "__v": 0
  },
  {
    "_id": "665f6c95980c13e83a4b2a7a",
    "user": "64d6a966a15d2e18ab96a263",
    "album": "64d2ae2308a725b72bd5c0cf",
    "timeListened": "2024-06-04T19:35:49.005Z",
    "__v": 0
  },
  {
    "_id": "665f6c55e919e2b68791e6a0",
    "user": "64d6a966a15d2e18ab96a263",
    "album": "64d2ae2308a725b72bd5c0cf",
    "timeListened": "2024-06-04T19:34:45.344Z",
    "__v": 0
  },
  {
    "_id": "665f6c4b1d5446711e80f8ff",
    "user": "64d6a966a15d2e18ab96a263",
    "album": "64d2ae2308a725b72bd5c0cf",
    "timeListened": "2024-06-04T19:34:35.985Z",
    "__v": 0
  },
  {
    "_id": "665f6bbf80e3baf26b58c508",
    "user": "64d6a966a15d2e18ab96a263",
    "album": "64d2ae2308a725b72bd5c0cf",
    "timeListened": "2024-06-04T19:32:15.571Z",
    "__v": 0
  },
  {
    "_id": "665f69c1150dc898bdd44267",
    "user": "64d6a966a15d2e18ab96a263",
    "album": "64d2ae2308a725b72bd5c0db",
    "timeListened": "2024-06-04T19:23:45.664Z",
    "__v": 0
  }
]
```